.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 coverage/
 npm-debug.log
 package-lock.json
+.idea
```

Comment:
We leave editor-specific ignores out of the package ignore file as users should just add them to their global git ignores.